#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 1.07kB done
#1 DONE 0.0s

#2 [internal] load metadata for docker.io/library/python:3.10
#2 DONE 0.0s

#3 [internal] load .dockerignore
#3 transferring context: 2B done
#3 DONE 0.0s

#4 [1/5] FROM docker.io/library/python:3.10
#4 DONE 0.0s

#5 [2/5] RUN mkdir -p ~/.pip && touch ~/.pip/pip.conf &&     echo "[global]" >> ~/.pip/pip.conf && echo "index-url=http://simple/" >> ~/.pip/pip.conf && echo "[install]" >> ~/.pip/pip.conf && echo "trusted-host=pypi..org" >> ~/.pip/pip.conf &&     apt-get update && apt-get install -y curl &&     curl -sSL https://install.python-poetry.org | python -
#5 CACHED

#6 [3/5] RUN pip install pytest pytest-xdist &&     pip install pipdeptree &&     git clone https://github.com/run-llama/llama_deploy.git &&     mkdir /repo &&     git config --global --add safe.directory /repo &&     cp -r /llama_deploy/. /repo && rm -rf /llama_deploy/ &&     rm -rf /llama_deploy &&     cd /repo && git checkout 47efff882cffa9b662f5a78e3032f5a13ea7bc17 &&     cd /repo && poetry install
#6 1.838 Looking in indexes: http://simple/
#6 2.010 Collecting pytest
#6 2.027   Downloading http://packages/pytest/pytest-8.3.4-py3-none-any.whl (343 kB)
#6 2.048      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 343.1/343.1 kB 23.9 MB/s eta 0:00:00
#6 2.125 Collecting pytest-xdist
#6 2.142   Downloading http://packages/pytest-xdist/pytest_xdist-3.6.1-py3-none-any.whl (46 kB)
#6 2.152      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 46.1/46.1 kB 6.0 MB/s eta 0:00:00
#6 2.241 Collecting tomli>=1
#6 2.264   Downloading http://packages/tomli/tomli-2.2.1-py3-none-any.whl (14 kB)
#6 2.321 Collecting pluggy<2,>=1.5
#6 2.337   Downloading http://packages/pluggy/pluggy-1.5.0-py3-none-any.whl (20 kB)
#6 2.402 Collecting packaging
#6 2.418   Downloading http://packages/packaging/packaging-24.2-py3-none-any.whl (65 kB)
#6 2.430      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.5/65.5 kB 9.0 MB/s eta 0:00:00
#6 2.458 Collecting iniconfig
#6 2.474   Downloading http://packages/iniconfig/iniconfig-2.0.0-py3-none-any.whl (5.9 kB)
#6 2.528 Collecting exceptiongroup>=1.0.0rc8
#6 2.546   Downloading http://packages/exceptiongroup/exceptiongroup-1.2.2-py3-none-any.whl (16 kB)
#6 2.607 Collecting execnet>=2.1
#6 2.625   Downloading http://packages/execnet/execnet-2.1.1-py3-none-any.whl (40 kB)
#6 2.637      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 40.6/40.6 kB 4.7 MB/s eta 0:00:00
#6 2.816 Installing collected packages: tomli, pluggy, packaging, iniconfig, execnet, exceptiongroup, pytest, pytest-xdist
#6 3.565 Successfully installed exceptiongroup-1.2.2 execnet-2.1.1 iniconfig-2.0.0 packaging-24.2 pluggy-1.5.0 pytest-8.3.4 pytest-xdist-3.6.1 tomli-2.2.1
#6 3.565 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#6 3.685 
#6 3.685 [notice] A new release of pip is available: 23.0.1 -> 25.0
#6 3.685 [notice] To update, run: pip install --upgrade pip
#6 4.497 Looking in indexes: http://simple/
#6 4.576 Collecting pipdeptree
#6 4.593   Downloading http://packages/pipdeptree/pipdeptree-2.25.0-py3-none-any.whl (32 kB)
#6 4.620 Requirement already satisfied: packaging>=24.1 in /usr/local/lib/python3.10/site-packages (from pipdeptree) (24.2)
#6 4.728 Collecting pip>=24.2
#6 4.743   Downloading http://packages/pip/pip-25.0-py3-none-any.whl (1.8 MB)
#6 4.772      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.8/1.8 MB 76.9 MB/s eta 0:00:00
#6 4.894 Installing collected packages: pip, pipdeptree
#6 4.894   Attempting uninstall: pip
#6 4.895     Found existing installation: pip 23.0.1
#6 5.070     Uninstalling pip-23.0.1:
#6 5.284       Successfully uninstalled pip-23.0.1
#6 6.684 Successfully installed pip-25.0 pipdeptree-2.25.0
#6 6.684 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#6 6.959 Cloning into 'llama_deploy'...
#6 9.592 Note: switching to '47efff882cffa9b662f5a78e3032f5a13ea7bc17'.
#6 9.592 
#6 9.592 You are in 'detached HEAD' state. You can look around, make experimental
#6 9.592 changes and commit them, and you can discard any commits you make in this
#6 9.592 state without impacting any branches by switching back to a branch.
#6 9.592 
#6 9.592 If you want to create a new branch to retain commits you create, you may
#6 9.592 do so (now or later) by using -c with the switch command. Example:
#6 9.592 
#6 9.592   git switch -c <new-branch-name>
#6 9.592 
#6 9.592 Or undo this operation with:
#6 9.592 
#6 9.592   git switch -
#6 9.592 
#6 9.592 Turn off this advice by setting config variable advice.detachedHead to false
#6 9.592 
#6 9.592 HEAD is now at 47efff8 fix: avoid early exit of RabbitMQ consumer (#397)
#6 10.65 Creating virtualenv llama-deploy-gW_DSdP6-py3.10 in /root/.cache/pypoetry/virtualenvs
#6 11.85 Installing dependencies from lock file
#6 12.67 
#6 12.67 Package operations: 87 installs, 0 updates, 0 removals
#6 12.67 
#6 12.67   - Installing typing-extensions (4.12.2)
#6 14.02   - Installing certifi (2024.8.30)
#6 14.02   - Installing charset-normalizer (3.4.0)
#6 14.02   - Installing exceptiongroup (1.2.2)
#6 14.03   - Installing frozenlist (1.5.0)
#6 14.03   - Installing h11 (0.14.0)
#6 14.03   - Installing idna (3.10)
#6 14.03   - Installing multidict (6.1.0)
#6 14.04   - Installing mypy-extensions (1.0.0)
#6 14.04   - Installing packaging (24.2)
#6 14.04   - Installing propcache (0.2.0)
#6 14.04   - Installing six (1.16.0)
#6 14.04   - Installing sniffio (1.3.1)
#6 14.05   - Installing urllib3 (2.2.3)
#6 16.48   - Installing aiohappyeyeballs (2.4.3)
#6 16.48   - Installing aiosignal (1.3.1)
#6 16.48   - Installing annotated-types (0.7.0)
#6 16.49   - Installing async-timeout (5.0.1)
#6 16.49   - Installing attrs (24.2.0)
#6 16.50   - Installing anyio (4.6.2.post1)
#6 16.50   - Installing greenlet (3.1.1)
#6 16.50   - Installing iniconfig (2.0.0)
#6 16.50   - Installing jmespath (1.0.1)
#6 16.51   - Installing httpcore (1.0.7)
#6 16.51   - Installing pamqp (3.3.0)
#6 16.52   - Installing pydantic-core (2.27.1)
#6 16.52   - Installing python-dateutil (2.9.0.post0)
#6 16.52   - Installing joblib (1.4.2)
#6 16.53   - Installing marshmallow (3.23.1)
#6 16.53   - Installing tomli (2.1.0)
#6 16.53   - Installing click (8.1.7)
#6 16.54   - Installing tqdm (4.67.1)
#6 16.54   - Installing typing-inspect (0.9.0)
#6 16.54   - Installing wrapt (1.17.0)
#6 16.55   - Installing smmap (5.0.1)
#6 16.55   - Installing regex (2024.11.6)
#6 16.55   - Installing requests (2.32.3)
#6 16.55   - Installing yarl (1.18.0)
#6 16.56   - Installing pluggy (1.5.0)
#6 16.56   - Installing types-awscrt (0.23.0)
#6 18.50   - Installing aiohttp (3.11.7)
#6 18.50   - Installing aioitertools (0.12.0)
#6 18.51   - Installing aiormq (6.8.1)
#6 18.51   - Installing botocore-stubs (1.35.68)
#6 18.51   - Installing coverage (7.6.8)
#6 18.52   - Installing botocore (1.35.36)
#6 18.52   - Installing dirtyjson (1.0.8)
#6 18.53   - Installing dataclasses-json (0.6.7)
#6 18.53   - Installing fsspec (2024.10.0)
#6 18.53   - Installing filetype (1.2.0)
#6 18.54   - Installing deprecated (1.2.15)
#6 18.54   - Installing httpx (0.27.2)
#6 18.55   - Installing networkx (3.4.2)
#6 18.55   - Installing nest-asyncio (1.6.0)
#6 18.55   - Installing numpy (2.1.3)
#6 18.56   - Installing pillow (11.0.0)
#6 18.56   - Installing nltk (3.9.1)
#6 18.56   - Installing gitdb (4.0.11)
#6 18.57   - Installing pyyaml (6.0.2)
#6 18.57   - Installing sqlalchemy (2.0.36)
#6 18.57   - Installing pytest (8.3.3)
#6 18.58   - Installing python-dotenv (1.0.1)
#6 18.58   - Installing types-aiobotocore-sns (2.15.2)
#6 18.58   - Installing starlette (0.41.3)
#6 18.59   - Installing tenacity (8.5.0)
#6 18.59   - Installing pydantic (2.10.1)
#6 18.60   - Installing tiktoken (0.8.0)
#6 18.60   - Installing types-aiobotocore-sqs (2.15.2)
#6 26.76   - Installing aio-pika (9.5.0)
#6 26.77   - Installing aiobotocore (2.15.2)
#6 26.77   - Installing aiokafka (0.11.0)
#6 26.77   - Installing asgiref (3.8.1)
#6 26.78   - Installing fastapi (0.115.5)
#6 26.78   - Installing gitpython (3.1.43)
#6 26.78   - Installing llama-index-core (0.12.0)
#6 26.79   - Installing kafka-python-ng (2.2.3)
#6 26.79   - Installing mypy (1.13.0)
#6 26.79   - Installing pydantic-settings (2.6.1)
#6 26.80   - Installing pytest-asyncio (0.23.8)
#6 26.80   - Installing pytest-cov (5.0.0)
#6 26.81   - Installing pytest-mock (3.14.0)
#6 26.81   - Installing python-multipart (0.0.18)
#6 26.81   - Installing redis (5.2.0)
#6 26.82   - Installing ruff (0.4.10)
#6 26.82   - Installing solace-pubsubplus (1.9.0)
#6 26.82   - Installing types-aiobotocore (2.15.2.post1)
#6 26.83   - Installing uvicorn (0.32.1)
#6 31.56 
#6 31.56 Installing the current project: llama-deploy (0.3.5)
#6 DONE 33.2s

#7 [4/5] RUN cd /repo && pytest --collect-only -q
#7 1.993 
#7 1.993 ==================================== ERRORS ====================================
#7 1.993 __________________________ ERROR collecting e2e_tests __________________________
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 <frozen importlib._bootstrap>:1050: in _gcd_import
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:1027: in _find_and_load
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:688: in _load_unlocked
#7 1.993     ???
#7 1.993 /usr/local/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:184: in exec_module
#7 1.993     exec(co, module.__dict__)
#7 1.993 e2e_tests/conftest.py:7: in <module>
#7 1.993     from llama_deploy import ControlPlaneConfig, SimpleMessageQueueConfig
#7 1.993 llama_deploy/__init__.py:4: in <module>
#7 1.993     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.993 llama_deploy/client/__init__.py:1: in <module>
#7 1.993     from .async_client import AsyncLlamaDeployClient
#7 1.993 llama_deploy/client/async_client.py:6: in <module>
#7 1.993     import httpx
#7 1.993 E   ModuleNotFoundError: No module named 'httpx'
#7 1.993 _______________________ ERROR collecting tests/apiserver _______________________
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 <frozen importlib._bootstrap>:1050: in _gcd_import
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:1027: in _find_and_load
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:688: in _load_unlocked
#7 1.993     ???
#7 1.993 /usr/local/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:184: in exec_module
#7 1.993     exec(co, module.__dict__)
#7 1.993 tests/apiserver/conftest.py:6: in <module>
#7 1.993     from fastapi.testclient import TestClient
#7 1.993 E   ModuleNotFoundError: No module named 'fastapi'
#7 1.993 __________________________ ERROR collecting tests/cli __________________________
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 <frozen importlib._bootstrap>:1050: in _gcd_import
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:1027: in _find_and_load
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:688: in _load_unlocked
#7 1.993     ???
#7 1.993 /usr/local/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:184: in exec_module
#7 1.993     exec(co, module.__dict__)
#7 1.993 tests/cli/conftest.py:4: in <module>
#7 1.993     from click.testing import CliRunner
#7 1.993 E   ModuleNotFoundError: No module named 'click'
#7 1.993 _____________________ ERROR collecting tests/client/models _____________________
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 <frozen importlib._bootstrap>:1050: in _gcd_import
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:1027: in _find_and_load
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
#7 1.993     ???
#7 1.993 <frozen importlib._bootstrap>:688: in _load_unlocked
#7 1.993     ???
#7 1.993 /usr/local/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:184: in exec_module
#7 1.993     exec(co, module.__dict__)
#7 1.993 tests/client/models/conftest.py:6: in <module>
#7 1.993     from llama_deploy.client import Client
#7 1.993 llama_deploy/__init__.py:4: in <module>
#7 1.993     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.993 llama_deploy/client/__init__.py:1: in <module>
#7 1.993     from .async_client import AsyncLlamaDeployClient
#7 1.993 llama_deploy/client/async_client.py:6: in <module>
#7 1.993     import httpx
#7 1.993 E   ModuleNotFoundError: No module named 'httpx'
#7 1.993 _________________ ERROR collecting tests/client/test_client.py _________________
#7 1.993 ImportError while importing test module '/repo/tests/client/test_client.py'.
#7 1.993 Hint: make sure your test modules/packages have valid Python names.
#7 1.993 Traceback:
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 tests/client/test_client.py:5: in <module>
#7 1.993     from llama_deploy.client import Client
#7 1.993 llama_deploy/__init__.py:4: in <module>
#7 1.993     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.993 llama_deploy/client/__init__.py:1: in <module>
#7 1.993     from .async_client import AsyncLlamaDeployClient
#7 1.993 llama_deploy/client/async_client.py:6: in <module>
#7 1.993     import httpx
#7 1.993 E   ModuleNotFoundError: No module named 'httpx'
#7 1.993 _____________ ERROR collecting tests/control_plane/test_config.py ______________
#7 1.993 ImportError while importing test module '/repo/tests/control_plane/test_config.py'.
#7 1.993 Hint: make sure your test modules/packages have valid Python names.
#7 1.993 Traceback:
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 tests/control_plane/test_config.py:5: in <module>
#7 1.993     from llama_deploy.control_plane import ControlPlaneConfig
#7 1.993 llama_deploy/__init__.py:4: in <module>
#7 1.993     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.993 llama_deploy/client/__init__.py:1: in <module>
#7 1.993     from .async_client import AsyncLlamaDeployClient
#7 1.993 llama_deploy/client/async_client.py:6: in <module>
#7 1.993     import httpx
#7 1.993 E   ModuleNotFoundError: No module named 'httpx'
#7 1.993 _____________ ERROR collecting tests/control_plane/test_server.py ______________
#7 1.993 ImportError while importing test module '/repo/tests/control_plane/test_server.py'.
#7 1.993 Hint: make sure your test modules/packages have valid Python names.
#7 1.993 Traceback:
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 tests/control_plane/test_server.py:5: in <module>
#7 1.993     from llama_deploy.control_plane import ControlPlaneConfig, ControlPlaneServer
#7 1.993 llama_deploy/__init__.py:4: in <module>
#7 1.993     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.993 llama_deploy/client/__init__.py:1: in <module>
#7 1.993     from .async_client import AsyncLlamaDeployClient
#7 1.993 llama_deploy/client/async_client.py:6: in <module>
#7 1.993     import httpx
#7 1.993 E   ModuleNotFoundError: No module named 'httpx'
#7 1.993 _________ ERROR collecting tests/message_queue_consumers/test_base.py __________
#7 1.993 ImportError while importing test module '/repo/tests/message_queue_consumers/test_base.py'.
#7 1.993 Hint: make sure your test modules/packages have valid Python names.
#7 1.993 Traceback:
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 tests/message_queue_consumers/test_base.py:5: in <module>
#7 1.993     from pydantic import PrivateAttr
#7 1.993 E   ModuleNotFoundError: No module named 'pydantic'
#7 1.993 __________ ERROR collecting tests/message_queues/test_apache_kafka.py __________
#7 1.993 ImportError while importing test module '/repo/tests/message_queues/test_apache_kafka.py'.
#7 1.993 Hint: make sure your test modules/packages have valid Python names.
#7 1.993 Traceback:
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 tests/message_queues/test_apache_kafka.py:6: in <module>
#7 1.993     from llama_deploy import QueueMessage
#7 1.993 llama_deploy/__init__.py:4: in <module>
#7 1.993     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.993 llama_deploy/client/__init__.py:1: in <module>
#7 1.993     from .async_client import AsyncLlamaDeployClient
#7 1.993 llama_deploy/client/async_client.py:6: in <module>
#7 1.993     import httpx
#7 1.993 E   ModuleNotFoundError: No module named 'httpx'
#7 1.993 ____________ ERROR collecting tests/message_queues/test_rabbitmq.py ____________
#7 1.993 ImportError while importing test module '/repo/tests/message_queues/test_rabbitmq.py'.
#7 1.993 Hint: make sure your test modules/packages have valid Python names.
#7 1.993 Traceback:
#7 1.993 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.993     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.993 tests/message_queues/test_rabbitmq.py:5: in <module>
#7 1.993     import aio_pika
#7 1.993 E   ModuleNotFoundError: No module named 'aio_pika'
#7 1.993 _____________ ERROR collecting tests/message_queues/test_redis.py ______________
#7 1.994 ImportError while importing test module '/repo/tests/message_queues/test_redis.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/message_queues/test_redis.py:7: in <module>
#7 1.994     from llama_deploy.message_consumers.base import BaseMessageQueueConsumer
#7 1.994 llama_deploy/__init__.py:4: in <module>
#7 1.994     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.994 llama_deploy/client/__init__.py:1: in <module>
#7 1.994     from .async_client import AsyncLlamaDeployClient
#7 1.994 llama_deploy/client/async_client.py:6: in <module>
#7 1.994     import httpx
#7 1.994 E   ModuleNotFoundError: No module named 'httpx'
#7 1.994 _____________ ERROR collecting tests/message_queues/test_simple.py _____________
#7 1.994 ImportError while importing test module '/repo/tests/message_queues/test_simple.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/message_queues/test_simple.py:5: in <module>
#7 1.994     from fastapi import HTTPException
#7 1.994 E   ModuleNotFoundError: No module named 'fastapi'
#7 1.994 ___________ ERROR collecting tests/message_queues/test_simple_app.py ___________
#7 1.994 ImportError while importing test module '/repo/tests/message_queues/test_simple_app.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/message_queues/test_simple_app.py:5: in <module>
#7 1.994     from fastapi.testclient import TestClient
#7 1.994 E   ModuleNotFoundError: No module named 'fastapi'
#7 1.994 ______ ERROR collecting tests/message_queues/test_simple_remote_client.py ______
#7 1.994 ImportError while importing test module '/repo/tests/message_queues/test_simple_remote_client.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/message_queues/test_simple_remote_client.py:6: in <module>
#7 1.994     import httpx
#7 1.994 E   ModuleNotFoundError: No module named 'httpx'
#7 1.994 _____________ ERROR collecting tests/message_queues/test_solace.py _____________
#7 1.994 ImportError while importing test module '/repo/tests/message_queues/test_solace.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/message_queues/test_solace.py:7: in <module>
#7 1.994     from solace.messaging.resources.topic import Topic
#7 1.994 E   ModuleNotFoundError: No module named 'solace'
#7 1.994 _______ ERROR collecting tests/orchestrators/test_simple_orchestrator.py _______
#7 1.994 ImportError while importing test module '/repo/tests/orchestrators/test_simple_orchestrator.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/orchestrators/test_simple_orchestrator.py:3: in <module>
#7 1.994     from llama_deploy.orchestrators.simple import SimpleOrchestrator
#7 1.994 llama_deploy/__init__.py:4: in <module>
#7 1.994     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.994 llama_deploy/client/__init__.py:1: in <module>
#7 1.994     from .async_client import AsyncLlamaDeployClient
#7 1.994 llama_deploy/client/async_client.py:6: in <module>
#7 1.994     import httpx
#7 1.994 E   ModuleNotFoundError: No module named 'httpx'
#7 1.994 ____________ ERROR collecting tests/services/test_agent_service.py _____________
#7 1.994 ImportError while importing test module '/repo/tests/services/test_agent_service.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/services/test_agent_service.py:1: in <module>
#7 1.994     from llama_index.core.llms import MockLLM
#7 1.994 E   ModuleNotFoundError: No module named 'llama_index'
#7 1.994 ____________ ERROR collecting tests/services/test_human_service.py _____________
#7 1.994 ImportError while importing test module '/repo/tests/services/test_human_service.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/services/test_human_service.py:6: in <module>
#7 1.994     from pydantic import PrivateAttr, ValidationError
#7 1.994 E   ModuleNotFoundError: No module named 'pydantic'
#7 1.994 _____________ ERROR collecting tests/services/test_tool_service.py _____________
#7 1.994 ImportError while importing test module '/repo/tests/services/test_tool_service.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/services/test_tool_service.py:5: in <module>
#7 1.994     from llama_index.core.tools import BaseTool, FunctionTool
#7 1.994 E   ModuleNotFoundError: No module named 'llama_index'
#7 1.994 ___________ ERROR collecting tests/services/test_workflow_service.py ___________
#7 1.994 ImportError while importing test module '/repo/tests/services/test_workflow_service.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/services/test_workflow_service.py:6: in <module>
#7 1.994     from llama_index.core.workflow import StartEvent, StopEvent, Workflow, step
#7 1.994 E   ModuleNotFoundError: No module named 'llama_index'
#7 1.994 __________ ERROR collecting tests/tools/test_agent_service_as_tool.py __________
#7 1.994 ImportError while importing test module '/repo/tests/tools/test_agent_service_as_tool.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/tools/test_agent_service_as_tool.py:5: in <module>
#7 1.994     from llama_index.core.llms import MockLLM
#7 1.994 E   ModuleNotFoundError: No module named 'llama_index'
#7 1.994 __________ ERROR collecting tests/tools/test_human_service_as_tool.py __________
#7 1.994 ImportError while importing test module '/repo/tests/tools/test_human_service_as_tool.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/tools/test_human_service_as_tool.py:7: in <module>
#7 1.994     from llama_deploy.message_queues.simple import SimpleMessageQueue
#7 1.994 llama_deploy/__init__.py:4: in <module>
#7 1.994     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.994 llama_deploy/client/__init__.py:1: in <module>
#7 1.994     from .async_client import AsyncLlamaDeployClient
#7 1.994 llama_deploy/client/async_client.py:6: in <module>
#7 1.994     import httpx
#7 1.994 E   ModuleNotFoundError: No module named 'httpx'
#7 1.994 ____________ ERROR collecting tests/tools/test_meta_service_tool.py ____________
#7 1.994 ImportError while importing test module '/repo/tests/tools/test_meta_service_tool.py'.
#7 1.994 Hint: make sure your test modules/packages have valid Python names.
#7 1.994 Traceback:
#7 1.994 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.994     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.994 tests/tools/test_meta_service_tool.py:3: in <module>
#7 1.994     from pydantic import PrivateAttr
#7 1.994 E   ModuleNotFoundError: No module named 'pydantic'
#7 1.994 _____________ ERROR collecting tests/tools/test_service_as_tool.py _____________
#7 1.995 ImportError while importing test module '/repo/tests/tools/test_service_as_tool.py'.
#7 1.995 Hint: make sure your test modules/packages have valid Python names.
#7 1.995 Traceback:
#7 1.995 /usr/local/lib/python3.10/importlib/__init__.py:126: in import_module
#7 1.995     return _bootstrap._gcd_import(name[level:], package, level)
#7 1.995 tests/tools/test_service_as_tool.py:3: in <module>
#7 1.995     from llama_deploy.message_queues.simple import SimpleMessageQueue
#7 1.995 llama_deploy/__init__.py:4: in <module>
#7 1.995     from llama_deploy.client import AsyncLlamaDeployClient, Client, LlamaDeployClient
#7 1.995 llama_deploy/client/__init__.py:1: in <module>
#7 1.995     from .async_client import AsyncLlamaDeployClient
#7 1.995 llama_deploy/client/async_client.py:6: in <module>
#7 1.995     import httpx
#7 1.995 E   ModuleNotFoundError: No module named 'httpx'
#7 1.995 =========================== short test summary info ============================
#7 1.995 ERROR e2e_tests - ModuleNotFoundError: No module named 'httpx'
#7 1.995 ERROR tests/apiserver - ModuleNotFoundError: No module named 'fastapi'
#7 1.995 ERROR tests/cli - ModuleNotFoundError: No module named 'click'
#7 1.995 ERROR tests/client/models - ModuleNotFoundError: No module named 'httpx'
#7 1.995 ERROR tests/client/test_client.py
#7 1.995 ERROR tests/control_plane/test_config.py
#7 1.995 ERROR tests/control_plane/test_server.py
#7 1.995 ERROR tests/message_queue_consumers/test_base.py
#7 1.995 ERROR tests/message_queues/test_apache_kafka.py
#7 1.995 ERROR tests/message_queues/test_rabbitmq.py
#7 1.995 ERROR tests/message_queues/test_redis.py
#7 1.995 ERROR tests/message_queues/test_simple.py
#7 1.995 ERROR tests/message_queues/test_simple_app.py
#7 1.995 ERROR tests/message_queues/test_simple_remote_client.py
#7 1.995 ERROR tests/message_queues/test_solace.py
#7 1.995 ERROR tests/orchestrators/test_simple_orchestrator.py
#7 1.995 ERROR tests/services/test_agent_service.py
#7 1.995 ERROR tests/services/test_human_service.py
#7 1.995 ERROR tests/services/test_tool_service.py
#7 1.995 ERROR tests/services/test_workflow_service.py
#7 1.995 ERROR tests/tools/test_agent_service_as_tool.py
#7 1.995 ERROR tests/tools/test_human_service_as_tool.py
#7 1.995 ERROR tests/tools/test_meta_service_tool.py
#7 1.995 ERROR tests/tools/test_service_as_tool.py
#7 1.995 !!!!!!!!!!!!!!!!!!! Interrupted: 24 errors during collection !!!!!!!!!!!!!!!!!!!
#7 1.995 no tests collected, 24 errors in 1.23s
#7 ERROR: process "/bin/sh -c cd /repo && pytest --collect-only -q" did not complete successfully: exit code: 2
------
 > [4/5] RUN cd /repo && pytest --collect-only -q:
1.995 ERROR tests/services/test_agent_service.py
1.995 ERROR tests/services/test_human_service.py
1.995 ERROR tests/services/test_tool_service.py
1.995 ERROR tests/services/test_workflow_service.py
1.995 ERROR tests/tools/test_agent_service_as_tool.py
1.995 ERROR tests/tools/test_human_service_as_tool.py
1.995 ERROR tests/tools/test_meta_service_tool.py
1.995 ERROR tests/tools/test_service_as_tool.py
1.995 !!!!!!!!!!!!!!!!!!! Interrupted: 24 errors during collection !!!!!!!!!!!!!!!!!!!
1.995 no tests collected, 24 errors in 1.23s
------
Dockerfile:20
--------------------
  18 |         cd /repo && git checkout 47efff882cffa9b662f5a78e3032f5a13ea7bc17 && \
  19 |         cd /repo && poetry install
  20 | >>> RUN cd /repo && pytest --collect-only -q
--------------------
ERROR: failed to solve: process "/bin/sh -c cd /repo && pytest --collect-only -q" did not complete successfully: exit code: 2
